### PR TITLE
When QField is pushing changes or synchronizing a cloud project, force view mode until done

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-1179 Check what happens if you sync a project, minimize the QFieldCloudPopup and create a new feature while syncing.](https://app.clickup.com/t/2192114/QF-1179)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 1881b830abdbd1f57827c8afdf40bd29784a0298:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2535-46c319256acea77a834d4125e00db64c9941b852-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2535-46c319256acea77a834d4125e00db64c9941b852-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2535-46c319256acea77a834d4125e00db64c9941b852-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2535-46c319256acea77a834d4125e00db64c9941b852-x86-android.apk)

File: src/qml/QFieldCloudPopup.qml [hunk 1]
```diff
@@ -16,8 +16,7 @@ Popup {
     header: PageHeader {
       title: qsTr('QFieldCloud')
 
-      showBackButton: cloudProjectsModel.currentProjectData.Status !== QFieldCloudProjectsModel.Uploading
-                        || cloudProjectsModel.currentProjectData.Status !== QFieldCloudProjectsModel.Downloading
```
Comment (suricactus) [inline src/qml/QFieldCloudPopup.qml:20]: Why you removed this part?

Comment (suricactus) [reply src/qml/QFieldCloudPopup.qml:20]: I guess you made the back button always visible since you prevented the user to edit

**Review by suricactus — APPROVED** [review]
It seems it prevents me to enter editing mode which is a nice first step. However, if I am already in ediitng of a feature the form is not closed and I can still save the feature. 

We can fix this either in this PR or in a follow-up, you decide.

I guess you made the back button always visible since you prevented the user to edit?

File: src/qml/QFieldCloudPopup.qml (hunk 1 above)
Comment (nirvn) [reply src/qml/QFieldCloudPopup.qml:20]: That's correct -- also, the condition was broken to begin with :) it should have been !download && !upload :)

**Comment by nirvn** [comment]
@suricactus , can you give some steps to reproduce?

**Comment by suricactus** [comment]
@nirvn I see you have already implemented what I was asking for in #2534 . Green light to merge!

File: src/qml/QFieldCloudPopup.qml (hunk 1 above)
Comment (suricactus) [reply src/qml/QFieldCloudPopup.qml:20]: clear, thanks

**Pull request merged**